### ISSUE TITLE
Launch on startup

### ISSUE THREAD

**Issue by n3h3m** [dexterleng/vimac-releases#41]
It's bit laborious to remember to start the Vimac upon startup. Most apps like BetterTouchTool, KarabinerElements do startup automatically. Would be cool if that thing works out of the box. 🙏

**Comment by dexterleng** [comment]
Yep working on it! Just a little bogged down with school assignments.

>
